### PR TITLE
mgr/crash: fix python3 invalid syntax problems

### PR DESCRIPTION
**Pull request by rjfd**
Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

File: src/pybind/mgr/crash/module.py [hunk 1]
```diff
@@ -45,13 +45,14 @@ def timestamp_filter(self, f):
         :param f: f(time) return true to keep crash report
         :returns: crash reports for which f(time) returns true
         """
-        def inner((_, meta)):
+        def inner(pair):
```
Comment (tchaikov) [inline src/pybind/mgr/crash/module.py]: nit,
why not put
```py
def inner(_, meta):
```

Comment (rjfd) [reply src/pybind/mgr/crash/module.py]: because only a single argument of type `tuple` is passed to the `inner` function.

Comment (tchaikov) [reply src/pybind/mgr/crash/module.py]: ahh, gotcha!

File: src/pybind/mgr/crash/module.py [hunk 2]
```diff
@@ -45,13 +45,14 @@ def timestamp_filter(self, f):
...
+            _, meta = pair
             meta = json.loads(meta)
             time = self.time_from_string(meta["timestamp"])
             return f(time)
         matches = filter(inner, six.iteritems(
             self.get_store_prefix("crash/")))
-        return map(lambda (k, m): (k, json.loads(m)), matches)
```
Comment (noahdesu) [inline src/pybind/mgr/crash/module.py]: thank you. i keep forgetting this feature was removed.

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-08-30_02:23:40-rados-wip-kefu-testing-2018-08-29-2346-distro-basic-smithi/

**Pull request merged**